config/core/configmaps/autoscaler.yaml

```diff
@@ -172,3 +172,29 @@ data:
     # See the algorithm here: http://bit.ly/38XiCZ3.
     # TODO(vagababov): tune after actual benchmarking.
     activator-capacity: "100.0"
+
+    # initial-scale is the cluster-wide default value for the initial deployment
+    # replica size of a revision.
+    # This value cannot be set to 0 while allow-zero-initial-scale is false.
+    # If annotation "autoscaling.knative.dev/initialScale" is not set,
+    # or "autoscaling.knative.dev/initialScale" is set to 0 while
+    # allow-zero-initial-scale is false, the cluster wide initial-scale value will
```

Comment:
so we permit to create a revision with autoscaling.knative.dev/initialScale==0 when feature is off, rather than just default with cluster wide value?

Reply:
no we don't permit creating a new revision with autoscaling.knative.dev/initialScale==0, but if there's an existing revision with initialScale 0, we will default with cluster wide value

Reply:
it doesn't make sense :-) Existing revision is exempt from these policies, since they only apply at creation :)

Reply:
ahh yeah that's true. so then we will just block new revision creation, and there won't be a problem with existing revisions.

Reply:
Here we are saying cluster wide value to override autoscaling.knative.dev/initialScale of a revision if autoscaling.knative.dev/initialScale==0 when allow-zero-initial-scale is false? But int `allow-zero-initial-scale` section we are saying new revision creation with autoscaling.knative.dev/initialScale==0 will not succeed if allow-zero-initial-scale is false. They are conflict.

Reply:
@yanweiguo yes you are correct. i haven't made the update to the PR yet, but `override autoscaling.knative.dev/initialScale of a revision if autoscaling.knative.dev/initialScale==0 when allow-zero-initial-scale is false` should be removed.